draft-ietf-suit-manifest.md

```diff
@@ -460,9 +460,9 @@ As required in {{Section 3 of RFC9019}} and as an extension of design goal 1, de
 
 The manifest processor must be resilient to these faults. In order to enable this resilience, systems implementing the manifest processor MUST make the following guarantees:
 
-Either:
+One of:
 1. A fallback/recovery image is provided so that a disrupted system can apply the SUIT Manifest again.
-2. Manifests are constructed so that repeated partial invocations of any manifest sequence always results in a correct system configuration.
```

Comment:
Is (2) essentially saying that the author needs to construct the manifest in such a case that the operations can be repeated without causing an incorrect system state? That is possible in some configurations but certainly not in all.